bool enable
uint32 state_machine_id
uint32 start_state_id
